torchvision/prototype/features/_image.py

```diff
@@ -32,6 +32,31 @@ def from_pil_mode(cls, mode: str) -> ColorSpace:
         else:
             return cls.OTHER
 
+    @staticmethod
+    def from_tensor_shape(shape: List[int]) -> ColorSpace:
+        return _from_tensor_shape(shape)
+
+
+def _from_tensor_shape(shape: List[int]) -> ColorSpace:
```

Comment:
```suggestion
def _from_tensor_shape(shape: Tuple[int, ...]) -> ColorSpace:
```

But I'm guessing this also doesn't work with JIT?

Reply:
JIT says no: 😄 
```
RuntimeError: 
Expression of type dots cannot be used in a type expression:
  File "./vision/torchvision/prototype/features/_image.py", line 40
def _from_tensor_shape(shape: Tuple[int, ...]) -> ColorSpace:
                                         ~~~ <--- HERE
```